guacamole-docker/README.md

```diff
@@ -12,8 +12,12 @@ How to use this image
 =====================
 
 Using this image will require an existing, running Docker container with the
-[guacd image](https://registry.hub.docker.com/u/glyptodon/guacd/), and another
-Docker container providing either a PostgreSQL or MySQL database.
+[guacd image](https://registry.hub.docker.com/u/glyptodon/guacd/), and either
+network access to a working LDAP server, or another Docker container providing
+a PostgreSQL or MySQL database.
+
+SQL Based Authentication
```

Comment:
If you do decided to keep it, I'd suggest rewording to "Database authentication". That would be consistent with the other existing docs (the manual), and consistent with the style of the other headers in this document (sentence case rather than title case).